src/App/Components/EditUI.js

```diff
@@ -0,0 +1,58 @@
+import React, { useContext, useRef, useState } from 'react';
+import { Context } from '../../Hooks/Context';
+import './EditUI.css';
+
+
+function EditUI({close}) {
+  const { userDetails, setUserDetails } = useContext(Context)
+  const  fullnameRef  = useRef(null)
+  const  phoneRef  = useRef(null)
+  const  emailRef  = useRef(null) 
+  const  imageRef  = useRef(null)
+  const [image, setImage] = useState('');
+  const [imageURL, setImageURL] = useState('') 
+
+  const handlePhotoUpload = (e) => {
+    e.preventDefault()
+    if (e.target.foodimage.files[0]) {
+       setImage(e.target.userimage.files[0]);
+       console.log(image);
```

Comment:
This image variable is used only within this function. It does not need to be a state.